teletext.go

```diff
@@ -867,6 +867,7 @@ func parseTeletextRow(i *Item, d decoder, fs func() styler, row []byte) {
 	var l = Line{}
 	var li = LineItem{InlineStyle: &StyleAttributes{}}
 	var started bool
+	started=true
```

Comment:
Mmm this is a huge change. Can you explain why you did that ? What's your use case ? (You can share a test file if that's easier to explain)